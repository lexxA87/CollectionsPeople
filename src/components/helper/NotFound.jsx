import React from "react";
import { Figure } from "react-bootstrap";

function NotFound() {
  return (
    <div className="text-center">
      <Figure>
        <Figure.Image
          width={300}
          height={300}
          alt="Not Found"
          src="/images/noSearchResult.jpg"
        />
        <Figure.Caption>
          <h1 className="display-6 text-warning">Not Found...</h1>
        </Figure.Caption>
      </Figure>
    </div>
  );
}

export default NotFound;
