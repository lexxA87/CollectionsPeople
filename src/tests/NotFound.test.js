import NotFound from "../components/helper/NotFound";
import ShallowRenderer from "react-test-renderer/shallow";

it("renders correctly NotFound component", () => {
  const renderer = new ShallowRenderer();
  renderer.render(<NotFound />);
  const view = renderer.getRenderOutput();

  expect(view).toMatchSnapshot();
});
