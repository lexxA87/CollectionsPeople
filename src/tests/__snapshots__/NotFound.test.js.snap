// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Loading component 1`] = `
<Spinner
  animation="border"
  role="status"
>
  <span
    className="visually-hidden"
  >
    Loading...
  </span>
</Spinner>
`;

exports[`renders correctly NotFound component 1`] = `
<div
  className="text-center"
>
  <Figure>
    <FigureImage
      alt="Not Found"
      fluid={true}
      height={300}
      src="/images/noSearchResult.jpg"
      width={300}
    />
    <FigureCaption>
      <h1
        className="display-6 text-warning"
      >
        noFound
      </h1>
    </FigureCaption>
  </Figure>
</div>
`;
